[DOCS] Add note on accessing defaults in FlexForm attributs (#3612)
TYPO3-Documentation/TYPO3CMS-Reference-CoreApi: Documentation/ApiOverview/FlexForms/Index.rst

   * :ref:`TypoScript: flexform <t3tsref:data-type-gettext-flexform>`

> .. index:: pair: FlexForms; Typoscript

Reviewer: Maybe "Default value" is better than TypoScript? And, it is not TypoScript but page TSconfig ;-)

.. _default-flexforms-attribute:

Providing default values for FlexForms attributes
-------------------------------------------------

When a new content element with an attached FlexForm is created, the 
default values for each FlexForm attribute is fetched from the
:xml:`<default>` XML attribute within the specification of each 
FlexForm attribute. If that is missing, an empty value will be 
shown in the backend (:ref:`FormEngine <FormEngine>`)
fields.

> While you can use TypoScript :ref:`t3tsref:pageTsTcaDefaults` to 

Reviewer: ```suggestion
While you can use page TSconfig's :ref:`t3tsref:pageTsTcaDefaults` to 
```

Sorry, haven't recognized that earlier.

.. index:: pair: FlexForms; Fluid
.. _read-flexforms-fluid:

Providing default values for FlexForms attributes

> ------------------------------------------------

Reviewer: ```suggestion
-------------------------------------------------
```

When a new content element with an attached FlexForm is created, the 
default values for each FlexForm attribute is fetched from the
:xml:`<default>` XML attribute within the specification of each 
FlexForm attribute. If that is missing, an empty value will be 
shown in the backend (:ref:`FormEngine <FormEngine>`)
fields.

> While you can use TypoScript :typoscript:`TCAdefaults` (see

Reviewer: I would link TCAdefaults directly, so no need for "(see ...)"

:ref:`TCAdefaults <TCAdefaults>`) to modify defaults of usual
TCA-based attributes, this is not possible on FlexForms. This
is because the values are calculated at an earlier step in

> the core workflow, where FlexForm values have not yet been

Reviewer: ```suggestion
the Core workflow, where FlexForm values have not yet been
```
There is a glossar how to write words, will provide the link later to you.
